Package.swift

```diff
@@ -28,19 +28,12 @@ let package = Package(
   ],
   dependencies: [
     .package(
-      name: "nanopb",
       url: "https://github.com/firebase/nanopb.git",
       "2.30908.0" ..< "2.30911.0"
```

Comment:
This may be out of sync with podspec dependency:
```ruby
s.dependency 'nanopb', '~> 3.30910.0'
```

Reply:
firebase-ios-sdk [specifies](https://github.com/firebase/firebase-ios-sdk/blob/856e201e5dbe0e7096a3f7a0a409deae0d79d8af/Package.swift#L151-L154):
```swift
    .package(
      url: "https://github.com/firebase/nanopb.git",
      "2.30909.0" ..< "2.30911.0"
    ),
```

Reply:
(Doesn't block this PR) But I think we should bump the Package.swift nanopb lower bound to `2.30910.0`, and maybe reconsider the ~> operator in the podspec.

Reply:
Any reason not to go ahead and release 3.30910.0 for both CocoaPods and SPM? Then we can uniformly update nanopb's clients for Firebase 11.

Reply:
Resolving. IIUC, `'nanopb', '~> 3.30910.0'` is the same as 'nanopb', '>= 3.30910.0', '< 3.30911.0'